src/pages/workspaces/workspace/Data.js

```diff
@@ -768,6 +787,8 @@ const WorkspaceData = _.flow(
                 wdsSchemaError && h(NoDataPlaceholder, {
                   message: 'Data tables are unavailable.'
                 }),
+                // TODO: Logic needs to slightly change here -- there is a delay when wdsSchema is updated
```

Comment:
mark for Slack update

Reply:
we can tackle the delay in a follow-on improvement ticket!

Reply:
Going to make a follow-up JIRA ticket!